website/components/AppWrapper.tsx

```diff
@@ -37,6 +38,7 @@ export default function AppWrapper(): ReactElement {
   const location = useLocation();
   const navigation = useNavigate();
 
+  const view3dRef = React.useRef<View3d | null>(null);
```

Comment:
`AppWrapper` now keeps a reference to `view3d` so it can be passed into `ShareModal` and the camera transform fetched on demand.